hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/util/CgroupsLCEResourcesHandler.java

```diff
@@ -138,13 +139,18 @@ void initConfig() throws IOException {
                 .NM_LINUX_CONTAINER_CGROUPS_STRICT_RESOURCE_USAGE,
             YarnConfiguration
                 .DEFAULT_NM_LINUX_CONTAINER_CGROUPS_STRICT_RESOURCE_USAGE);
+    this.cappedMultiplier =
```

Comment:
Some suggestions for style:
```suggestion
    this.cappedMultiplier = conf.getFloat(
```